### PR TITLE
change icon app

### PR DESCRIPTION
**Pull request by vu9596**
change icon app

### CODE REVIEW

**Comment by vu9596** [comment]
ready

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: need an endline

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
 </adaptive-icon>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: need an endline

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by vu9596** [comment]
ready

**Pull request merged**